src/test/java/jep/test/TestPyBuiltins.java

```diff
@@ -0,0 +1,336 @@
+            return false;
+        }
+        if (builtins.isinstance(subtypeOne, subtypeTwo)){
+            failure = "issubclass builtin returns unrelated subtype is a subclass";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testList() {
+        PyObject listBuiltin = builtins.list();
+        PyObject listGetValue = interp.getValue("[]", PyObject.class);
+        if (!listBuiltin.equals(listGetValue)){
+            failure = "list builtin does not return an empty list";
+            return false;
+        }
+        listBuiltin = builtins.list(new String[] {"1", "2", "3"});
+        listGetValue = interp.getValue("['1','2','3']", PyObject.class);
+        if (!listBuiltin.equals(listGetValue)){
+            failure = "list builtin does not return a full list";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testObject() {
+        PyObject o = builtins.object();
+        if (!o.equals(object)){
+            failure = "object builtin does not return an empty object";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testSet() {
+        PyObject setBuiltin = builtins.set();
+        PyObject setGetValue = interp.getValue("set()", PyObject.class);
+        if (!setBuiltin.equals(setGetValue)){
+            failure = "set builtin does not return an empty set";
+            return false;
+        }
+        setBuiltin = builtins.set(new String[] {"1", "2", "3"});
+        setGetValue = interp.getValue("{'1','2','3'}", PyObject.class);
+        if (!setBuiltin.equals(setGetValue)){
+            failure = "set builtin does not return a full set";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testSetAttr() {
+        builtins.setattr(subobject, "name", "value");
+        if (!"value".equals(subobject.getAttr("name"))){
+            failure = "setattr builtin does not set value";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testTuple() {
+        PyObject tupleBuiltin = builtins.tuple();
+        PyObject tupleGetValue = interp.getValue("()", PyObject.class);
+        if (!tupleBuiltin.equals(tupleGetValue)){
+            failure = "tuple builtin does not return an empty tuple";
+            return false;
+        }
+        tupleBuiltin = builtins.tuple(new String[] {"1", "2", "3"});
+        tupleGetValue = interp.getValue("('1','2','3')", PyObject.class);
+        if (!tupleBuiltin.equals(tupleGetValue)){
+            failure = "tuple builtin does not return a full tuple";
+            return false;
+        }
+        return true;
+    }
+
+    private boolean testType() {
+        if (!objectType.equals(builtins.type(object))){
+            failure = "type builtin returns wrong type for an object";
+            return false;
+        }
+        return true;
+    }
```

Comment:
Your tests are good but they mostly test for expected behavior.  Should some testing test unexpected behavior, like you pass the wrong type of argument in and get an exception?  I'm ok with it if you don't want to do that.